### PR TITLE
Personal dms reminder

### PR DESCRIPTION
**Pull request by khalford**
Changed the `PostToDMs.run()` method to take a list of users rather than single user. This allows us to send multiple message threads without needing to pull all the GitHub PRs again in the class init step.
If we run this feature as is multiple times, we will get an asynchronous timeout from the main app hence the changes.
Also checks if a user is in the workspace before trying to send the message.
Adding this to the scheduling for Mondays at 9:00 in favour of using @ to mention people

### CODE REVIEW

File: cloud-chatops/docker-compose.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 services:
   cloud_chatops:
-    image: harbor.stfc.ac.uk/stfc-cloud/cloud-chatops:1.0.6
+    image: harbor.stfc.ac.uk/stfc-cloud/cloud-chatops:1.1.6
```
Comment (DavidFair) [inline cloud-chatops/docker-compose.yaml:3]: Nit, according to SEMVER we should reset the patch version on a minor bump
https://semver.org/#spec-item-7

**Review by DavidFair — CHANGES_REQUESTED** [review]

File: cloud-chatops/docker-compose.yaml (hunk 1 above)
Comment (khalford) [reply cloud-chatops/docker-compose.yaml:3]: Yes, my mistake

**Review by DavidFair — APPROVED** [review]

File: cloud-chatops/src/features/base_feature.py [hunk 2]
```diff
@@ -115,6 +115,15 @@ def _format_prs(prs: List[PrData]) -> List[PrData]:
...
 
+    def validate_user(self, user: str) -> None:
+        """Validate that the given user is in the workspace."""
+        try:
+            self.client.users_profile_get(user=user)
+        except SlackApiError as exc:
+            raise UserNotFound(
+                f"The user with member ID {user} is not in this workspace."
+            ) from exc
+
```
Comment (meoflynn) [inline cloud-chatops/src/features/base_feature.py:126]: Codecov has raised a warning about a missing unit test, could we add a unit test to check this error is raised?

**Review by meoflynn — CHANGES_REQUESTED** [review]

File: cloud-chatops/src/features/base_feature.py (hunk 2 above)
Comment (khalford) [reply cloud-chatops/src/features/base_feature.py:126]: That is covered [here](https://github.com/stfc/cloud-docker-images/blob/4ac5e6de365e2713e8d90939027358f1ebf5a280/cloud-chatops/tests/test_base_feature.py#L178) I'm not sure why code cov is complaining. @DavidFair do you have any ideas?

**Review by meoflynn — APPROVED** [review]

**Pull request merged**